molgenis-data-rest/src/main/java/org/molgenis/data/rest/v2/EntityCollectionBatchRequestV2.java

```diff
@@ -17,6 +17,7 @@
 	public abstract List<Map<String, Object>> getEntities();
 
 	@Override
+	@SuppressWarnings("squid:S1206")
```

Comment:
To discuss with the team: would it be usefull to always accompany these suppressions with a comment stating what the identifier means?